DOCS-#3609: Mention OmniSci engine in root README.md (#3683)
modin-project/modin: README.md

### Installation

#### From PyPI

> Modin can be installed using following line:

Reviewer: ```suggestion
Modin can be installed with `pip`:
```

Modin will automatically detect which engine you have installed and use that for
scheduling computation!

#### From conda-forge

> Following line will install modin and 3 engines ([Ray](https://github.com/ray-project/ray),

Reviewer: ```suggestion
Installing from conda forge using `modin-all` will install Modin and 3 engines: ([Ray](https://github.com/ray-project/ray),
```

[Dask](https://github.com/dask/dask) and [Omnisci](https://modin.readthedocs.io/en/latest/UsingOmnisci/index.html))

```bash
conda install -c conda-forge modin-all
```

> Also each engine could be installed explicitly. For example this line installs all 3:

Reviewer: ```suggestion
Each engine can also be installed individually:
```

```bash

> conda install -c conda-forge modin-ray modin-dask modin-omnisci

Reviewer: ```suggestion
conda install -c conda-forge modin-ray  # install Ray engine with Modin
conda install -c conda-forge modin-dask  # install Dask engine with Modin
conda install -c conda-forge modin-omnisci  # install Omnisci engine with Modin
```

import modin.pandas as pd
```

> Check [complete Modin documentation](https://modin.readthedocs.io/en/latest/UsingOmnisci/index.html) for Omnisci engine setup

Reviewer 2: ```suggestion
Check [this Modin section](https://modin.readthedocs.io/en/latest/UsingOmnisci/index.html) for Omnisci engine setup
```
